components/ui/SmartFileInput.vue

```diff
@@ -36,6 +36,10 @@ export default {
       type: String,
       default: null,
     },
+    maxSize: {
+      type: String,
```

Comment:
nb, as I think this works regardless:

Is there a reason this is `String` (as opposed to `Number`)?